core/test/view_index_test.cpp

```diff
@@ -106,30 +132,8 @@ TEST_F(ViewIndex2DTest, increment_2D_transpose) {
   }
 }
 
-TEST_F(ViewIndex2DTest, increment_1D) {
```

Comment:
So the old behavior was to wrap, and we don't do this any more, so the tests are removed?

Reply:
No. I removed them because I had initially removed the ability to broadcast with ViewIndex. I restored the tests now.